plugins/org.eclipse.sirius.tests.junit.support/src/org/eclipse/sirius/tests/support/api/SiriusDiagramTestCase.java

```diff
@@ -1300,8 +1300,13 @@ protected boolean setDDiagramAttribute(final EditingDomain domain, final DDiagra
     protected void unsynchronizeDiagram(final DDiagram diagram) {
         // Unsynchronized the diagram
         TransactionalEditingDomain domain = session.getTransactionalEditingDomain();
-        Command setUnsynschronizedCmd = SetCommand.create(domain, diagram, DiagramPackage.Literals.DDIAGRAM__SYNCHRONIZED, false);
-        domain.getCommandStack().execute(setUnsynschronizedCmd);
+        domain.getCommandStack().execute(new RecordingCommand(domain) {
```

Comment:
The modification of this class is no longer needed (with the modifications proposed in the test class). You can revert it.

For information, the "setUnsynschronizedCmd" was not executable because the aird resource was in read only (not copied in the workspace).